app/build.gradle

```diff
@@ -7,7 +7,7 @@ android {
 
     defaultConfig {
         applicationId "com.pensive.android.romplanuib"
-        minSdkVersion 21
```

Comment:
Do we really want to increase this to version 24?
According to https://developer.android.com/about/dashboards/ about 40% still uses versions 21-23.